hw/ip/rv_timer/data/rv_timer_testplan.hjson

```diff
@@ -67,5 +67,56 @@
       stage: V2
       tests: ["rv_timer_stress_all"]
     }
+    {
+      name: chip_sw_timer
+      desc: '''Verify the timeout interrupt assertion.
+
+            - Configure the RV_TIMER to generate interrupt after a set timeout.
+            - Issue a WFI to wait for the interrupt to trigger.
+            - Service the interrupt when it triggers; verify that it came from rv_timer.
+            - Verify that the interrupt triggered only after the timeout elapsed.
+            '''
+      stage: V2
+      si_stage: SV2
+      features: ["RV_TIMER.ENABLE", "RV_TIMER.CONFIG", "RV_TIMER.COMPARE", "RV_TIMER.INTERRUPT",
+                 "RV_TIMER.COUNTER"]
+      tests: ["chip_sw_rv_timer_irq"]
+    }
+    {
+      name: tick_configuration
+      desc: '''Verify that the timer can be configured to generate a system tick.
+
+            - Configure to generate a tick of `T` microseconds long.
```

Comment:
```suggestion
            - Configure the creation of a tick of `T` microseconds long.
```

Reply:
I'm sure this represents what I mean, I changed to `Configure the timer to generate a tick...` let me know what you think.

Reply:
I think I agree. (I understand your suggestion, @johngt, but that actually means something else)